cmd/abapEnvironmentPullGitRepo.go

```diff
@@ -92,9 +80,12 @@ func runAbapEnvironmentPullGitRepo(options *abapEnvironmentPullGitRepoOptions, t
 	return err
 }
 
-func pullRepositories(repositories []abaputils.Repository, pullConnectionDetails abaputils.ConnectionDetailsHTTP, client piperhttp.Sender, pollIntervall time.Duration) (err error) {
-	log.Entry().Infof("Start cloning %v repositories", len(repositories))
+func pullRepositories(repositories []abaputils.Repository, pullConnectionDetails abaputils.ConnectionDetailsHTTP, client piperhttp.Sender, pollIntervall time.Duration, ignoreCommit bool) (err error) {
+	log.Entry().Infof("Start pulling %v repositories", len(repositories))
 	for _, repo := range repositories {
+		if ignoreCommit {
+			repo.CommitID = ""
+		}
```

Comment:
See above. This check could be done before without passing the `ignoreCommit` to the function again